### PR TITLE
Add the "Travel to PyCon India" page on the website.

### LINKED ISSUE #91: Add a venue page describing the venue
This should be some what similar to https://us.pycon.org/2018/venue/ . Approach the local volunteers for details regarding the venue for accommodation details etc. You can contact them in discord channel.

### PR DESCRIPTION
**Pull request by aswanipranjal**
This issue adds details about how to get to PyCon (travelling to pycon) page and updates the link of that page on other HTML pages.

Fixes #91

### CODE REVIEW

**Comment by aswanipranjal** [comment]
Continued from #181. Please finish merging #150 so that I can make the appropriate changes and this can proceed further.

File: travel-2018.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+  </nav>
+
+
+  <!-- Section: about -->
+
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
```
Comment (realslimshanky) [inline travel-2018.html]: It would be better with `PyCon India`. What do you guys think? @ananyo2012 @vipulgupta2048

Comment (vipulgupta2048) [reply travel-2018.html]: Yes, its more specific

Comment (realslimshanky) [reply travel-2018.html]: cc: @aswanipranjal

Comment (vipulgupta2048) [reply travel-2018.html]: @aswanipranjal Make it. 
`How to get to PyCon India 2018?` or `Getting to PyCon India 2018`

These would be more appropriate. Choose whichever you prefer.

**Comment by vipulgupta2048** [comment]
@aswanipranjal Added some changes to the content through a commit, and left a commit message for you. Do review the changes made in https://github.com/pythonindia/inpycon2018/pull/182/commits/577fafc754a63ea6851498aa284242d50d5efcd8

**Comment by ananyo2012** [comment]
I updated the content a bit which is as per my local experience in travelling in Hyderabad. @aswanipranjal @vipulgupta2048 Thanks a lot for compiling the content. And if you find any problem in Traveling to Venue do reach out to me :smile: Merging this.

**Pull request merged**